qt/settingsdialog.py

```diff
@@ -614,11 +614,14 @@ def __init__(self, parent):
             "<b>{}:</b> {}".format(
                 _("Warning"),
```

Comment:
What do you think about such a modification? That wording is IMHO less negative.

```suggestion
                _('Attention'),
```

Reply:
How about using the standard log severity levels (eg. just "Info")?

Reply:
```suggestion
                _('Info'),
```